src/utils/toaster.ts

```diff
@@ -0,0 +1,15 @@
+import { toast, ToastType } from "react-hot-toast";
+
+const toastMap: Record<ToastType, (message: string | JSX.Element) => string> = {
```

Comment:
it shouldn't be a union, the message will always be a string

Reply:
what is the Record for?

Reply:
> it shouldn't be a union, the message will always be a string

I added it just incase, i will change it to justs string

Reply:
> what is the Record for?

I used it because the handler function outputed from toast[type] did not really have a specific type, so i used the record to separate out the different handler fucntions